scripts/deploy_oak_functions_loader

```diff
@@ -20,17 +20,22 @@ docker build \
 
 docker push "${FUNCTIONS_DOCKER_IMAGE_NAME}:latest"
 
-# Build and push an individual example application built on Oak Functions.
```

Comment:
Should this file be moved to `oak_functions/examples/hello_world` since it is very `hello_world` specific? Or perhaps it should be made parametric to be usable for other examples too, in a separate PR.

Reply:
Yeah I was thinking about that too, currently each example would need its own Dockerfile anyways, so I'm not sure how much value there is in parametrizing the script only. I need to read more on how to create docker images in different ways perhaps, then it may be easier to parametrize everything, either with scripts or in runner.